README.md

```diff
@@ -243,4 +240,8 @@ for a container-centric world
 system 
 * [Docker Compose](https://github.com/docker/compose) (formerly Fig): 
 Define and run multi-container apps
+* [Kitematic](https://github.com/kitematic/kitematic): The easiest way to use Docker on a Mac
```

Comment:
add docker/distribution while you're at it maybe since you'll amend for DCO anyway?